compiler/src/dmd/cparse.d

```diff
@@ -5702,6 +5715,59 @@ final class CParser(AST) : Parser!AST
                             }
                             break;
 
+                        case TOK.identifier:
+                        {
+                            /* check for compiler builtins macros/ or specifier references.
+                             * this is a workaround to avoid trying to resolve macros that
+                             * reference other identifiers or functions
+                             * so D users should avoid using custom macros beginning with __
+                             * to reference identifers
```

Comment:
Why workaround? GDC knows what all these built-ins are so is not in the best interest to skip them.